api/cmd/portainer/main.go

```diff
@@ -212,12 +214,27 @@ func main() {
 		}
 	}
 
-	if *flags.AdminPassword != "" {
-		log.Printf("Creating admin user with password hash %s", *flags.AdminPassword)
+	AdminPasswordHash := ""
```

Comment:
The `AdminPasswordHash` variable should be `adminPasswordHash`

Reply:
Yup, this makes sense